### PR TITLE
Braze Cohorts Public Documentation

### PR DESCRIPTION
**Pull request by Innovative-GauravKochar**
### Proposed changes
This PR adds docs for the Braze Cohorts destination. The Braze Cohorts destination goes to Public Beta March 7.

### Merge timing
- Merge and deploy during Tues, March 7 docs deploy.

### Related issues (optional)
[Jira Ticket](https://segment.atlassian.net/jira/software/projects/HGI/boards/512?assignee=63617339fc0cc7a600b03c6b&selectedIssue=HGI-367)

### CODE REVIEW

**Comment by markzegarelli** [comment]
Hi @Innovative-GauravKochar!

Can you remove the updates the yaml files from this PR? For destinations in Private Beta, we need to use a different mechanism to add any necessary metadata. Once the yaml updates are removed, I will update the PR.

File: src/_data/catalog/destinations.yml [hunk 1]
```diff
@@ -9550,6 +9550,217 @@ items:
         '@path': $.traits
       _update_existing_only: false
     trigger: type = "identify"
+- id: 63872c01c0c112b9b4d75412
```
Comment (kdaswani) [inline src/_data/catalog/destinations.yml:9553]: @markzegarelli I think you normally put together destinations.yml yourself.  Is that right? Feel free to remove this file if it causes issues. The main thing is just getting the public doc together and uploaded next week.

**Review by kdaswani — APPROVED** [review]
Made some updates and @Innovative-GauravKochar removed the yml files. Approved by me! Passing to docs team @markzegarelli @stayseesong for final review before March 7 docs deploy.

FYI I will be OOO March 1-6 so please feel free to commit any changes you see fit and merge this in ahead of the March 7 deploy. Thanks!

**Comment by kdaswani** [comment]
@markzegarelli is this good to go for tomorrow's deploy?

**Comment by markzegarelli** [comment]
@kdaswani Yep! I'll merge this and the others today, to ensure they are out tomorrow morning.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**